xyz-psql-connector/src/main/java/com/here/xyz/psql/DatabaseHandler.java

```diff
@@ -165,50 +171,51 @@ void reset() {
     @Override
     protected synchronized void initialize(Event event) {
```

Comment:
Instead of synchronizing the method itself, wouldn't be better to synchronize both reset() and initialize() methods' bodies by the same object?